source/_integrations/binary_sensor.knx.markdown

```diff
@@ -8,21 +8,13 @@ ha_iot_class: Local Push
 ha_domain: knx
 ---
 
-<div class='note'>
-  
-The `knx` integration must be configured correctly to use this integration, see [KNX Integration](/integrations/knx).
-
-</div>
```

Comment:
I've removed this note as the platforms can't be configured without first configuring knx: now since https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md